src/test/java/com/softserveinc/ita/SearchTest.java

```diff
@@ -0,0 +1,37 @@
+package com.softserveinc.ita;
+
+import com.softserveinc.ita.pageobjects.HomePage;
+import com.softserveinc.ita.pageobjects.SearchResultPage;
+import com.softserveinc.ita.utils.runners.TestRunner;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SearchTest extends TestRunner {
+
+    @Test
+    public void verifySearchResultsTest() {
+        var homePage = new HomePage();
+        var requiredItem = "Xiaomi Redmi Note 10";
+        homePage
```

Comment:
init here var searchResultPage =
because performSearch returns object of this page

Reply:
like this ?
var searchResultPage =  homePage
                .closeAdvertisingBannerIfDisplayed()
                .setTextInSearchBar(requiredItem)
                .performSearch();